### ISSUE TITLE
layout issue in Holoscape, and otherwise?

### ISSUE THREAD

**Issue by Connoropolous** [holochain-open-dev/community-mutual-credit#12]
<img width="1377" alt="Screen Shot 2020-05-19 at 11 11 31 AM" src="https://user-images.githubusercontent.com/1409121/82355815-bbba0880-99d0-11ea-87ec-3016df4bcf18.png">

**Comment by guillemcordoba** [comment]
Yeah thanks... Holoscape is weird in that sense, but layout was not one of my priorities, I'll add this to the next iteration.
